backend/src/routers/annotation_router.py

```diff
@@ -124,11 +127,75 @@ def upload_annotation_section(
         "data_set": section_name,
         "data_source": "rosalution-manual",
         "version": str(date.today()),
-        "value": str(new_file_object_id),
+        "value": {"file_id": str(new_file_object_id), "created_date": str(datetime.now())},
     }
 
-    repositories['genomic_unit'].annotate_genomic_unit_with_file(genomic_unit, annotation_unit)
+    try:
+        repositories['genomic_unit'].annotate_genomic_unit_with_file(genomic_unit, annotation_unit)
+    except Exception as exception:
+        raise HTTPException(status_code=500, detail=str(exception)) from exception
 
     response.status_code = status.HTTP_201_CREATED
 
     return {'section': section_name, 'image_id': str(new_file_object_id)}
+
+
+@router.post("/{genomic_unit}/update/image/{file_id}")
+def update_annotation_image(
+    file_id: str,
+    genomic_unit: str,
+    genomic_unit_type: GenomicUnitType = Form(...),
+    section_name: str = Form(...),
```

Comment:
After a bit of discussion and planning, the `section_name` references throughout the full stack process has been renamed to `data_set`. This has a lot of implications across many files, but the result should be cleaner overall.

Second, the URL's have been updated to follow the `'/{genomic_unit/{data_set}/update/{file_id}'` styling. This gives a lot more description for how the routes are being handled. It describe which `genomic_unit` is being updated and which `data_set` it's being updated for.

Third, the `'Gene Homology/Multi-Sequence Alignment'` data_set was a great example as it contains a slash. If this were to be passed in the route above, it would become incorrect and not found as the slash would turn it into a new route entirely. 

Given the issue with the slash, the annotation rendering configuration has been updated to de-couple the section header's name with the actual dataset being annotated:

```javascript
{
    'type': 'section',
    'class': '',
    'header': 'Gene Homology/Multi-Sequence Alignment',
    'anchor': 'Gene_Homology',
    'header_datasets': [],
    'props': {
      'attachmentDataset': 'GeneHomology_Multi-SequenceAlignment',
      'genomicAttachmentType': 'hgvs_variant',
    },
    'rows': [{
      'class': '',
      'datasets': [
        {
          'dataset': 'GeneHomology_Multi-SequenceAlignment',
          'genomicType': 'hgvs_variant',
          'type': 'images-dataset',
        },
      ],
    }],
  }, 
```

The above object now contains a new dataset name that is sanitized and is URL safe.